recipe/meta.yaml

```diff
@@ -25,35 +25,37 @@ build:
 requirements:
   host:
     - python
-    - debugpy >=1.0.0
-    - hatchling >=1.4
-    - ipython >=7.23.1
-    - jupyter_client >=6.1.12
-    - packaging
     - pip
-    - comm >=0.1
-    - psutil
+    - setuptools
     - wheel
+    - debugpy
+    - comm
+    - hatchling
+    - ipython
+    - jupyter_client
+    - packaging
+    - psutil
   run:
-    - appnope  # [osx]
-    - comm >=0.1.1
-    - debugpy >=1.0
+    - python
+    - debugpy >=1.6.5
     - ipython >=7.23.1
+    - comm >=0.1.1
+    - traitlets >=5.4.0
     - jupyter_client >=6.1.12
```

Comment:
The upstream has a  runtime "jupyter_core>=4.12,!=5.0.*", constraint. Is jupyter forge really just a test time dependency or is it needed for runtime as well?

Reply:
It should most likely be runtime instead of tests. Not sure what conda-forge is doing here https://github.com/conda-forge/ipykernel-feedstock/blob/d5e8001e203cf2097e076debf20a67fb58ad2a49/recipe/meta.yaml#L75